tests/app/data/test_schema_validation.py

```diff
@@ -0,0 +1,38 @@
+from app.schema_loader import schema_loader
+import jsonschema
+import logging
+import unittest
+import os
+
+
+logger = logging.getLogger(__name__)
+
+
+class TestSchemaValidation(unittest.TestCase):
+
+    def test_schema(self):
+
+      errors = []
+
+      for file in schema_loader.available_schemas():
+
+        try:
+          filename = os.path.splitext(file)[0]
+
+          eq_id = filename.split("_")[0]
+
+          json_schema = schema_loader.load_schema(eq_id, filename[len(eq_id)+1:])
```

Comment:
I think it would be clearer if we loaded the schema file directly here, rather than extract the eq_id and form_type and use the load_schema method.